### DISCUSSION TITLE
Firewall dapat dikenal sebagai sistem keamanan jaringan

### DISCUSSION THREAD

**Discussion by sunshinebecomesyou** [svecx/Flow-Docs#11] [Q&A] [answered]
akses jaringan dan melindungi sistem dari serangan dan ancaman dari internet.

**Comment by svecx** [comment, marked as answer]
Jaringan Intern: Ini adalah jaringan yang berada di dalam organisasi atau perusahaan, dan meliputi semua sistem dan perangkat yang terhubung ke jaringan.

Jaringan Ekstern: Ini adalah jaringan internet atau jaringan luar organisasi yang mencoba mengakses jaringan intern.
